spring-cloud-alibaba-sentinel-gateway/src/main/java/com/alibaba/cloud/sentinel/gateway/GatewayEnvironmentPostProcessor.java

```diff
@@ -0,0 +1,59 @@
+package com.alibaba.cloud.sentinel.gateway;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.env.EnvironmentPostProcessor;
+import org.springframework.core.env.*;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by zhh on 2019/8/17.
```

Comment:
the author format is not correct.

Reply:
Sorry. I have added the license and author.